sql/createClient.sql

```diff
@@ -0,0 +1,15 @@
+-- 3. As a member of the Sales team I want to be able to create a new client. 
+-- I should be able to store a name, address and phone number
+
+CREATE TABLE client (
+    client_id int PRIMARY KEY AUTO_INCREMENT,
+    f_name varchar(255),
+    l_name varchar(255),
```

Comment:
Has there been an assumption here that a client will be a person, what about if they are company?